web/src/components/organisms/Project/Content/ContentDetails/hooks.ts

```diff
@@ -0,0 +1,100 @@
+import { useCallback, useMemo } from "react";
+
+import { FieldType } from "@reearth-cms/components/molecules/Schema/types";
+import {
+  SchemaFiledType,
+  useCreateItemMutation,
+  useUpdateItemMutation,
+} from "@reearth-cms/gql/graphql-client-api";
+
+import useContentHooks from "../hooks";
+
+type Props = {
+  itemId: string | undefined;
+};
+
+export default ({ itemId: itemID }: Props) => {
+  const { currentModel, itemsData } = useContentHooks();
+
+  const [createNewItem] = useCreateItemMutation({
+    refetchQueries: ["GetItems"],
+  });
+
+  const handleItemCreate = useCallback(
+    async (data: {
+      schemaID: string;
+      fields: { schemaFieldID: string; type: FieldType; value: string }[];
+    }) => {
+      if (!data.schemaID) return;
```

Comment:
This is redundant because data AND schemaID are mandatory

Reply:
You are right, fixed.